### PR TITLE
Squirrel

### PR DESCRIPTION
**Pull request by arnovr**
Squirrels are members of the family Sciuridae, a family that includes small or medium-size rodents. The squirrel family includes tree squirrels, ground squirrels, chipmunks, marmots (including woodchucks), flying squirrels, and prairie dogs amongst other rodents. Squirrels are indigenous to the Americas, Eurasia, and Africa, and were introduced by humans to Australia.[1] The earliest known squirrels date from the Eocene period and are most closely related to the mountain beaver and to the dormouse among other living rodent families.

### CODE REVIEW

**Comment by arnovr** [comment]
@rosstuck @lucasvanlierop something wrong with docker login?

**Comment by lucasvanlierop** [comment]
Looks awesome!

Somehow there's an issue with PR's from non members, curently there are 3 PR's with the same issue while mine build normally. I've asked Travis for help:
https://twitter.com/lucasvanlierop/status/781413257550626816

File: arnovr-squirrel/raffler.sq [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                return ascii[uppercase].toupper();
+	}
+	if ( char == SPACE ) {
+		return " ";
+	}
+	return "";
+}
+
+local names = file(fileName,"rb");
+local data = names.readblob(1000);
```
Comment (arnovr) [inline arnovr-squirrel/raffler.sq]: Just figured, this probably doesnt work with alot of names

Comment (arnovr) [inline arnovr-squirrel/raffler.sq]: Will check tonight or something

**Comment by rjkip** [comment]
@lucasvanlierop They respond to e-mails pretty quickly, don't know about Twitter. Anyhoo, I found this in [their documentation](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings):

> Similarly, we do not provide these values to untrusted builds, triggered by pull requests from another repository.

They've thought things through :wink: Don't know how we'd fix this situation, though. Just merge and :pray: for now, I guess.

**Comment by arnovr** [comment]
What @rjkip said, merge it :+1:

**Comment by lucasvanlierop** [comment]
Can you try to update to make use of @rjkip's fix?

**Comment by arnovr** [comment]
That seems to work :)

**Pull request merged**